account_creation_page.py

```diff
@@ -0,0 +1,21 @@
+import streamlit as st
+
+
+with st.form("account_creation_form"):
+
+    st.header("Social Stock Analyzer")
+    st.subheader("Create your Account")
+
+    Username = st.text_input('',placeholder='Username')
+    Password = st.text_input('',placeholder='Password')
+    Confirm_password = st.text_input('',placeholder='Confirm password')
+    Email = st.text_input('',placeholder='E-mail address')
```

Comment:
We also need a field for first and last name.

Reply:
Ok, I will add them.